apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ComposeView.kt

```diff
@@ -563,7 +590,12 @@ fun ComposeView(
         sent = send(chat, MsgContent.MCText(msgText), quotedItemId, null, live, ttl)
       }
     }
+    val wasForwarding = cs.forwarding
     clearState(live)
+    val draft = chatModel.draft.value
+    if (wasForwarding && chatModel.draftChatId.value == chat.chatInfo.id && draft != null) {
+      composeState.value = draft
+    }
```

Comment:
a couple of interactions seems incorrect:
- when forwarding to a chat where draft is available, draft is offered as "forward comment" (already in compose), but if it had an attachment it won't be sent. instead draft shouldn't be included initially, but after forward is sent compose is populated with draft
- second is not related to draft - when compose is not empty, forward in the same chat overwrites it, making text empty and removing preview. instead text and link preview should be kept (other previews should be removed)